Client/Frontend/Intro/IntroViewController.swift

```diff
@@ -31,14 +31,14 @@ struct IntroViewControllerUX {
     
     static let CardTitleWelcome = NSLocalizedString("Intro.Slides.Welcome.Title", tableName: "Intro", value: "Thanks for choosing Firefox!", comment: "Title for the first panel 'Welcome' in the First Run tour.")
     static let CardTitleSearch = NSLocalizedString("Intro.Slides.Search.Title", tableName: "Intro", value: "Your search, your way", comment: "Title for the second  panel 'Search' in the First Run tour.")
-    static let CardTitlePrivate = NSLocalizedString("Intro.Slides.Private.Title", tableName: "Intro", value: "Browse like no one’s watching", comment: "Title for the third panel 'Private Browsing' in the First Run tour.")
+    static let CardTitlePrivate = NSLocalizedString("Intro.Slides.Private.Title", tableName: "Intro", value: "Browse like no one's watching", comment: "Title for the third panel 'Private Browsing' in the First Run tour.")
```

Comment:
Agreed we should use an apostrophe type here, but I think this is the character: ’ (should be a curly apostrophe pasted there)

Reply:
Wait, that looks just like the one in the left. I have to keep copy-pasting these into a text editor to see the characters, they all look too similar in this diff view...

The one on the left is a quote in Character Palette on OS X. However, the replaced one isn't a typographic apostrophe –or whatever the curly apostrophe is called.

Reply:
Fixed all straight quotes to be RIGHT SINGLE QUOTATION MARK.